lib/Driver/ToolChains.cpp

```diff
@@ -307,6 +307,7 @@ void ToolChain::addCommonFrontendArgs(const OutputInfo &OI,
   // Pass on file paths that should be remapped in debug info.
   inputArgs.AddAllArgs(arguments, options::OPT_debug_prefix_map);
   inputArgs.AddAllArgs(arguments, options::OPT_coverage_prefix_map);
+  inputArgs.AddAllArgs(arguments, options::OPT_file_prefix_map);
```

Comment:
I believe `AddAllArgs` actually takes both an `ArrayRef` and also up to 3 specifiers which we could use instead.

Reply:
Yep, done